styles/pages/_homepage.scss

```diff
@@ -1,47 +0,0 @@
-.ipaddressapp {
```

Comment:
try using tailwind for css

Reply:
I can very fluent with SCSS so can we use both.